server/database/models/books.js

```diff
@@ -16,6 +16,10 @@ module.exports = (sequelize, DataTypes) => {
       foreignKey: 'authorId',
       onDelete: 'CASCADE'
     });
+
+    Books.hasMany(models.Fines, {
+      as: 'id'
```

Comment:
Remove the 'as' property. Just specify the foreign key and the onDelete constraint here.